app/bundles/FormBundle/Helper/FormFieldHelper.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
- * @copyright   2014 Mautic Contributors. All rights reserved
+ * @copyright   2018 Mautic Contributors. All rights reserved
```

Comment:
Actually, the copyrights dates should be set to the current date only for the newly created files. Not for the old ones. The year should state when the file was created and so when the copyright started.

Reply:
I know I just make mistake ;)Revert